components/match2/wikis/smite/match_group_input_custom.lua

```diff
@@ -114,6 +115,27 @@ function CustomMatchGroupInput.processPlayer(player)
 	return player
 end
 
+--
+-- function to check for draws
+--
+---@param scoreTable table[]
+---@return boolean
+function CustomMatchGroupInput.placementCheckDraw(scoreTable)
```

Comment:
why are you adding this?
is it used somewhere?
and it isn't mentioned in the PRs description

Reply:
Method should not have been added, was something another user added to the dev module that got mistakenly copied over